components-api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -50,7 +50,7 @@ case object MandatoryParameterValidator extends ParameterValidator {
 
 case object NotBlankParameterValidator extends ParameterValidator {
 
-  private final lazy val blankStringLiteralPattern: Pattern = Pattern.compile("'\\s*'")
+  private final lazy val blankStringLiteralPattern: Pattern = Pattern.compile("['\"]\\s*['\"]")
```

Comment:
Shouldn't we check that the closing quote is the same as the opening one?

Reply:
Totally agree. Regex should look like this:
```
"(['\"])\\s*\\1"
```

Reply:
regex stays as is 😄 we have common agreement that we validate only expressions that are possible to compile